src/App.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from './test-utils';
+import userEvent from '@testing-library/user-event';
+import { createMemoryHistory } from 'history';
+import { BrowserRouter, Router, MemoryRouter } from 'react-router-dom';
+
+import App from './App';
+
+describe('React Router', () => {
+  it('Renders Home Page', () => {
+    render(
+      <BrowserRouter>
```

Comment:
why BrowserRouter in this test?

Reply:
We found that when BrowserRouter wasn't there it would fail the test, but when wrapped around it passed, and it is because BrowserRouter isn't wrapped within App.tsx but wrapped within Index.tsx